velox/docs/functions/spark/datetime.rst

```diff
@@ -85,6 +85,15 @@ These functions support TIMESTAMP and DATE input types.
     ``day`` need to be from 1 to 31, and matches the number of days in each month.
     days of ``year-month-day - 1970-01-01`` need to be in the range of INTEGER type.
 
+.. spark:function:: months_between(timestamp1, timestamp2, roundOff) -> double
+
+    Returns number of months between times ``timestamp1`` and ``timestamp2``.
+    If ``timestamp1`` is later than ``timestamp2``, the result is positive.
+    If ``timestamp1`` and ``timestamp2`` are on the same day of month,
+    or both are the last day of month, returns, time of day will be ignored.
+    Otherwise, the difference is calculated based on 31 days per month.
+    The result is rounded to 8 decimal places if ``roundOff`` is set to true.
```

Comment:
is this '8'? What does that mean? An example would be helpful. What happens if roundOff is false?

Reply:
@mbasmanova done